app/jobs/clock/inbound_webhooks_retry_job.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Clock
+  class InboundWebhooksRetryJob < ApplicationJob
+    include SentryCronConcern
+
+    queue_as "clock"
+
+    def perform
+      InboundWebhook
+        .where(status: ["pending", "processing"])
```

Comment:
Hmmmm, I think the retry should be a bit more laid back, if we do this every 15 minutes, when we have a large backlog, the queue will be quickly filled with jobs no?

Reply:
we could use the unique config to ensure the jobs are unique or touch the records here after queueing them to refresh the updated_at and avoid the job being queued more than once...

The logic behind "shorter" times to retry is that we expect webhooks to be something immediate... wait for 24 hours is too long. What do you think?

Reply:
yes, agreed, web hooks should be processed quickly. Perhaps it makes sense to store a retry count on the web hook? And we can skip web hooks that have ben retried an amount of time?